webapp/src/core.ts

```diff
@@ -11,6 +11,8 @@ import * as pkg from "./package";
 import Cloud = pxt.Cloud;
 import Util = pxt.Util;
 
+export let highContrast: boolean;
```

Comment:
It allow us to display the #msg in high contrast (the temporary banner that says for example "building .hex file"). As this DOM is outside of #root, we had to pass the setting to core when it displays the banners.